.travis.yml

```diff
@@ -38,7 +42,10 @@ env:
     - TASK=test:shared
 matrix:
   allow_failures:
+    - env: TASK=coverage:agent
     - env: TASK=coverage:finance
```

Comment:
ℹ️for reference: manually produced coverage stats for `coverage:finance`:

* Finance 
```
--------------|----------|----------|----------|----------|----------------|
File          |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
--------------|----------|----------|----------|----------|----------------|
 contracts/   |    99.38 |    92.65 |     97.3 |      100 |                |
  Finance.sol |    99.38 |    92.65 |     97.3 |      100 |                |
--------------|----------|----------|----------|----------|----------------|
All files     |    99.38 |    92.65 |     97.3 |      100 |                |
--------------|----------|----------|----------|----------|----------------|
```